src/shadowbox/server/manager_service.ts

```diff
@@ -165,7 +169,7 @@ export class ShadowsocksManagerService {
       logging.error(error);
       if (error instanceof errors.InvalidPortNumber) {
         return next(new restify.InvalidArgumentError(error.message));
-      } else if (error instanceof errors.PortInUse) {
+      } else if (error instanceof errors.PortUnavailable) {
```

Comment:
Talked offline. We'll use
- ConflictError (409) for port already in use
- InvalidContentError (400) for invalid port number.